api/mesh/v1alpha1/mesh.proto

```diff
@@ -75,6 +75,10 @@ message Mesh {
   // Constraints that applies to the mesh and its entities
   Constraints constraints = 7;
 
+  // List of policies to skip creating by default when the mesh is created.
+  // e.g. TrafficPermission, MeshRetry, etc.
```

Comment:
Add info that `*` means skip all policies

Reply:
Resolved